cmd/compose/handlers/handler.go

```diff
@@ -21,13 +23,13 @@ type Profiles interface {
 }
 
 type Users interface {
-	GetUser(ctx context.Context, id string) (usersapi.User, error)
```

Comment:
Well, you see the interface before was incapsulating the transport behind. Could you try to keep it that way?

Reply:
How to achieve that? For the HTTP we created a package in `compose/pkg/usersapi`. You could do the same, go `compose/pkg/usersgrpc` and create there a structure that implements `Users interface` from this package and encapsulate the transport. Then inject another dependency from the `main.go`. By these means, the current file would be left unchanged.

Reply:
you are right, changed :+1: